projects/laji/src/app/+project-form/project-form.component.ts

```diff
@@ -131,20 +136,24 @@ export class ProjectFormComponent implements OnInit, OnDestroy {
       startWith(this.router.url)
     );
 
-    this.showNav$ = routerEvents$.pipe(
+    this.subscriptionScreen = this.browserService.lgScreen$.subscribe(value => this.isDesktopScreen = value );
+
+    routerEvents$.pipe(
       mergeMap((url: string) =>
         form$.pipe(
           map(form =>
             !(
               (!form.options?.useNamedPlaces && url.match(/\/form$/))
               || (form.options?.useNamedPlaces && url.match(/\/places\/MNP\.\d+$/))
               || (url.match(/\/form\/(.*\/)?((JX\.)|(T:))\d+$/))
-              || (form.options?.useNamedPlaces && (url.match(/\/form\/places/) || url.match(/\/form\/MHL.*\/places/)))
+              || (!this.isDesktopScreen)
             )
           )
         )
       )
-    );
+    ).subscribe(showNav => {
+      this.showNav$ = showNav !== this.openSidebar ? of(showNav) : of(this.openSidebar);
+    });
```

Comment:
There seems to be some confusion here about how RXJS works. An observable (`showNav$`) should compose it's output value from piping other observables. Reinitializing the whole observable is going to cause problems (here it is initalized at `ngOnInit` and later on at `checkToggledSidebar`). 

In order to get `showNav$` to output the right values, the proper approach is to turn all the data you need for composing the right value into observables (or if they are observables already like the desktop screen check is, use those) and use those as input observables. So instead of binding the `isDesktopScreen` into a variable, use it as an input observable for `showNav$`, like this:

```
this.showNav$ = combineLatest(this.browserService.lgScreen$, routerEvents$).pipe(
  mergeMap([isDeskopScreen, url]) =>
         form$.pipe(
          map(form =>
            !(
              (!form.options?.useNamedPlaces && url.match(/\/form$/))
              || (form.options?.useNamedPlaces && url.match(/\/places\/MNP\.\d+$/))
              || (url.match(/\/form\/(.*\/)?((JX\.)|(T:))\d+$/))
              || (isDesktopScreen)
            )
          )
        )
  )
)
```

To elaborate why, here are some reasons:
1.You can't in theory be sure that `this.isDesktopScreen` is initialized when you initialize `showNav$`. When using the desktop screen check as an input for the observable, you can be certain that it's initialized.
2. If the screen size changes, updating `this.isDesktopScreen` won't trigger checking new value for `showNav$`.

Reply:
Ok